server/boot/passport.js

```diff
@@ -1,7 +1,7 @@
 module.exports = function (server) {
   var router = server.loopback.Router();
   router.get('/auth/success', function (req, res, next) {
-    res.redirect(require('../config').baseUrl + '/#/passport/');
+    res.redirect(require('../config').appUrl + '/#/passport/');
```

Comment:
What was the change to `appUrl` for? `config.example.json` has not been updated, are we meant to rename `baseUrl` to `appUrl`?